template/Cargo.toml

```diff
@@ -5,13 +5,19 @@ authors = ["[your_name] <[your_email]>"]
 edition = "2018"
 
 [dependencies]
-ink_abi = { git = "https://github.com/paritytech/ink", package = "ink_abi", default-features = false, features = ["derive"], optional = true }
-ink_primitives = { git = "https://github.com/paritytech/ink", package = "ink_primitives", default-features = false }
-ink_core = { git = "https://github.com/paritytech/ink", package = "ink_core", default-features = false }
-ink_lang = { git = "https://github.com/paritytech/ink", package = "ink_lang", default-features = false }
+ink_abi = { version = "2.0.0", git = "https://github.com/paritytech/ink", tag = "latest-v2", package = "ink_abi", default-features = false, features = ["derive"], optional = true }
```

Comment:
Version should be just "2" then. Otherwise it does not match the expected semver of the moving tag.